docs/modules/ROOT/pages/documentation/extensions/notifiers/README.adoc

```diff
@@ -0,0 +1,29 @@
+= Notifier Extensions
+
+The notifier extensions allow notifications from the Notification Service to propagate to various channels on top of the notifiers provided by default.
+
+You can find the code for each notifier here: https://github.com/payara/Notifiers. That codebase includes an example notifier that can be used as a base for any additional notifiers you might want.
```

Comment:
Better to state that "The codebase illustrates a notifier example in case you might want to write your own extension if an specific notifier is not supported"